### ISSUE TITLE
remove mouseout from category selector?

### ISSUE THREAD

**Issue by jread-usgs** [USGS-VIZLAB/water-use-15#346]
I think the "preview" category mouseover may result in confusion. Instead, mousing over a category could be the same as clicking the category (it sets the active category).

**Comment by aappling-usgs** [comment]
Wow, I was completely blind to this possibility for confusion before, but makes sense now that I've played with it on dev with this proposal in mind - I agree that it'd be more intuitive for the category to stay on whichever one you hovered over most recently.

**Comment by jread-usgs** [comment]
And does removing the mouse out event also help us resolve the iOS tap issue?

**Comment by lindsaycarr** [comment]
@jread-usgs I don't think it would help with that directly, but we're going to implement this as part of the solution for that.

**Comment by lindsaycarr** [comment]
@jread-usgs should be fixed, but we didn't get a chance to push to beta to check. Probably can do that on 5/25 since Mindi no longer needs beta to be stable.

**Comment by aappling-usgs** [comment]
Should be on beta now - @jread-usgs , could you check it out on your iOS?

**Comment by jread-usgs** [comment]
The category selector is fixed, but tapping a county does not highlight it.

**Comment by aappling-usgs** [comment]
Hmm, I can reproduce that on my phone and on my desktop with mobile developer tools, too. I think I can fix this - PR coming soon.
